eo-maven-plugin/src/test/java/org/eolang/maven/tojos/ForeignTojosTest.java

```diff
@@ -116,8 +120,67 @@ void findsAnyTojoIfSeveralTojosWithTheSameIdWereAdded() {
         );
     }
 
+    @ParameterizedTest
+    @MethodSource("tojoFunctions")
+    void throwsExceptionIfKeyWasNotFoundInTojo(
+        final String key,
+        final Func<ForeignTojo, Object> method) {
+        final ForeignTojo tojo = this.tojos.add("string");
+        Assertions.assertThrows(
+            IllegalStateException.class,
+            () -> method.apply(tojo),
+            String.format("Should throw an exception if key='%s' was not found in Tojo", key)
+        );
+        final IllegalStateException thrown = Assertions.assertThrows(
+            IllegalStateException.class,
+            tojo::verified
+        );
+        Assertions.assertEquals(
+            "There is no 'VERIFIED' attribute in the tojo",
+            thrown.getMessage(),
+            "Should throw an exception if key 'VERIFIED' was not found in Tojo"
+        );
+    }
+
+    @Test
+    void findsKeyInTojo() {
```

Comment:
@Yanich96 I think this test should be renamed with something like `doesNotThrowsAnException` and also use `@MethodSource` annotation.

Reply:
@maxonfjvipon Do I need to check all attributes in this test?